### ISSUE TITLE
Validation accuracy always 100?

### ISSUE THREAD

**Issue by AlphonsG** [facebookresearch/Mask2Former#18]
Hello,

Thank you for this project and code. I'm running a custom semantic segmentation training job (based on [this](https://github.com/facebookresearch/Mask2Former/blob/main/configs/ade20k/semantic-segmentation/swin/maskformer2_swin_large_IN21k_384_bs16_160k_res640.yaml) config)  with one class (custom class 'AT') and for some reason my validation accuracy after an epoch is always 100:

[12/21 21:20:26 d2.evaluation.evaluator]: Total inference time: 0:00:44.703675 (0.065935 s / iter per device, on 1 devices)
[12/21 21:20:26 d2.evaluation.evaluator]: Total inference pure compute time: 0:00:37 (0.055430 s / iter per device, on 1 devices)
[12/21 21:20:26 d2.evaluation.sem_seg_evaluation]: OrderedDict([('sem_seg', {'mIoU': 100.0, 'fwIoU': 100.0, 'IoU-AT': 100.0, 'mACC': 100.0, 'pACC': 100.0, 'ACC-AT': 100.0})])
[12/21 21:20:26 d2.engine.defaults]: Evaluation results for ade20k_full_sem_seg_val in csv format:
[12/21 21:20:26 d2.evaluation.testing]: copypaste: Task: sem_seg
[12/21 21:20:26 d2.evaluation.testing]: copypaste: mIoU,fwIoU,mACC,pACC
**[12/21 21:20:26 d2.evaluation.testing]: copypaste: 100.0000,100.0000,100.0000,100.0000**

When I view the loss curves in tensorboard it seems like the model is learning so I'm not sure what's going on:

![2021-12-21 21_35_27-localhost_5139 - Remote Desktop Connection](https://user-images.githubusercontent.com/45003643/146939713-d982e726-b778-4fd1-abbf-fc03c9a9c651.png)

Here's the full config:

[config.zip](https://github.com/facebookresearch/Mask2Former/files/7755716/config.zip)

Any ideas?

Thank you

**Comment by bowenc0221** [comment]
You only have 1 class and the model will have 100% accuracy by always predicting that class.

**Comment by sergiev** [comment]
@bowenc0221 what is the proper way to train single class (except background, of course) semantic segmentation model?

**Comment by lxr-1204** [comment]
I often encounter this problem, do you have any solutions? i will be very grateful to you

**Comment by AlphonsG** [comment]
@lxr-1204 hello, I stopped using this repo but it was a very long time ago, not sure how to fix
